servicemesh-install/topics/install-remove.adoc

```diff
@@ -1,19 +1,48 @@
-[[removing_service-mesh]]
+[[removing-service-mesh]]
 = Removing {ProductName}
 
-The following command removes the {ProductShortName} from an existing installation. Log in as a cluster admin and run these commands from any host with access to the cluster.
+The following process removes the {ProductShortName} from an existing installation. Log in as a cluster admin and run these commands from any host with access to the cluster.
 
-. Remove the custom resource by running the following command:
+[[remove-control-plane]]
+== Remove the control plane
+
+. Retrieve the name of the installed custom resource with this command:
 +
 
 ----
-$ oc delete -n istio-operator installation istio-installation
+oc get controlplanes -n istio-system
 ----
 
 +
-. Remove the operator by running the following command:
-+
+. Remove the custom resource by running the following command:
+
+----
+$ oc delete -n istio-system -f <name_of_custom_resource>
+----
+
+[NOTE]
+====
+When you remove the custom resource, {ProductShortName} tells the operator to begin uninstalling everything it installed.
+====
+
+
+[[remove-operator]]
+== Remove the operator
+Remove the operator by running the following commands:
+
+----
+$ oc delete -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.10/deploy/istio_v1alpha1_installation_crd.yaml
+$ oc delete -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.10/deploy/istio_v1alpha3_controlplane_crd.yaml
+$ oc delete -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.10/deploy/operator.yaml
+$ oc delete -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.10/deploy/rbac.yaml
+$ oc delete -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.10/deploy/service_account.yaml
```

Comment:
BTW, all the links above are wrong now but the operator.

Reply:
The yamls have been trimmed down into one servicemesh-operator.yaml link that works.

Reply:
This is corrected.